File: Algoritmos e Logica de Programacao/Java/ExerciciosJavaVetores/src/application/ProblemaNegativos.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaNegativos {

	public static void main(String[] args) {
		/*
		 * Problema "negativos"
Faça um programa que leia um número inteiro positivo N (máximo = 10) e depois N números inteiros
e armazene-os em um vetor. Em seguida, mostrar na tela todos os números negativos lidos.
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int n =sc.nextInt();
		int vet[] = new int[n];
		for(int i=0;i<n;i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextInt();
		}
		
		System.out.println("NUMEROS NEGATIVOS: ");
		for(int i=0;i<n;i++) {
			if(vet[i]<0) {
				System.out.println(vet[i]);
			}
		}
		
		
		sc.close();
	}

}

File: Algoritmos e Logica de Programacao/C/Vetores/ProblemaNegativos.c
/*
Problema "negativos"
Faça um programa que leia um número inteiro positivo N (máximo = 10) e depois N números inteiros
e armazene-os em um vetor. Em seguida, mostrar na tela todos os números negativos lidos.
*/
#include<stdio.h>
#include<stdlib.h>
int main (){

	int n =0,i=0;
	int vet[10];
	int numero = 0;
	printf("Quantos numeros voce vai digitar? ");
	scanf("%d",&n);
	for(i=0;i<n;i++){
		printf("Digite um numero: ");
		scanf("%d",&vet[i]);
	}
	
	printf("NUMEROS NEGATIVOS: \n");
	
	for(i=0;i<n;i++){
		if(vet[i]<0){
			printf("%d\n",vet[i]);
		}
	}
	
    return 0;
}
